clients/web/test/spec/userSpec.js

```diff
@@ -96,6 +96,16 @@ describe('Create an admin and non-admin user', function () {
         }, 'front page to display');
     });
 
+    it('check the user count with an admin user', function () {
+        girderTest.login('admin', 'Admin', 'Admin', 'adminpassword!')();
+        girderTest.goToUsersPage()();
+        runs(function () {
```

Comment:
You can move this entire block (really, just the assertion on line 104) up to line 93. There's no reason for another whole test spec.